src/combat/planner/context.py

```
@@ -67,6 +67,25 @@ def has_strict_route(self) -> bool:
+        通过此公共方法安全关闭 route。
+        Returns:
+            True 如果有 route 被关闭，False 如果没有活动 route。
+        """
+
+        route = self._state.locked_route
+        if route is None:
+            return False
+        try:
+            route.close()
+        except Exception:
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)